node/heartbeat/hearbeatMessageInfo.go

```diff
@@ -30,49 +35,28 @@ func (hbmi *heartbeatMessageInfo) clockTime() time.Time {
 
 // Sweep updates all records
 func (hbmi *heartbeatMessageInfo) sweep() {
-	for _, phb := range hbmi.peerHeartbeats {
-		crtDuration := hbmi.getTimeHandler().Sub(phb.TimeStamp)
-		phb.IsActive = crtDuration < hbmi.maxDurationPeerUnresponsive
-		if phb.MaxInactiveTime.Duration < crtDuration {
-			phb.MaxInactiveTime.Duration = crtDuration
-		}
+	crtDuration := hbmi.getTimeHandler().Sub(hbmi.timeStamp)
+	hbmi.isActive = crtDuration < hbmi.maxDurationPeerUnresponsive
+	if hbmi.maxInactiveTime.Duration < crtDuration {
+		hbmi.maxInactiveTime.Duration = crtDuration
 	}
 }
 
 // HeartbeatReceived processes a new message arrived from a p2p address
```

Comment:
change the comment as it does not match what the function does.

Reply:
done